cmd/ps_test.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetShipmentEndpointHttp(t *testing.T) {
```

Comment:
Just realized you meant the other function (`getShipmentPrimaryPort`) rather than `TestGetShipmentEndpointHttp`.